content/04.network_applications.md

```diff
@@ -40,18 +40,44 @@ In the future, it may be beneficial for these models is to incorporate other typ
 
 ### Unifying Applications
 
-Knowledge graphs have been used in many biomedical applications ranging from identifying protein functions [@doi:10.1186/s12859-018-2163-9] to prioritizing cancer genes [@doi:10.1093/bioinformatics/bty148] to recommending safer drugs to patients [@arxiv:1710.05980; @doi:10.1609/aaai.v33i01.33011126].
+Knowledge graphs have been used in many biomedical applications ranging from identifying protein functions [@doi:10.1186/s12859-018-2163-9] to prioritizing cancer genes [@doi:10.1093/bioinformatics/bty148] to recommending safer drugs to patients [@arxiv:1710.05980; @doi:10.1609/aaai.v33i01.33011126] (Figure {@fig:unifying_applications}).
 In this section we discuss how knowledge graphs are being applied in biomedical settings. 
 We put particular emphasis on an emerging set of techniques: those that project knowledge graphs into a low dimensional space.
 
-#### Disease and Gene Interactions
-
-1. Mention disease gene prioritization
-2. Mention Disease gene associations
-
-#### Protein Protein Interactions
-
-1. Mention predicting genes interacting genes
+![
+Overview of biomedical applications that make use of knowledge graphs.
+Categories consist of: (a) Multi-Omic applications, (b) Pharmaceutical Applications and (c) Clinical Applications.
+](images/figures/unifying_applications_overview.png){#fig:unifying_applications}
+
+#### Multi-Omic Applications
+
+Multi-omic applications consist of studying genes (genomics), RNA (transcriptomics) and proteins (proteomics).
+Approaches in this category use knowledge graphs to establish connections between biological entities (genes, RNA, proteins) themselves and diseases via tasks such as gene prioritization [@doi:10.1093/jamia/ocy117], predicting interactions between proteins [@doi:10.1089/cmb.2012.0273;10.1186/1752-0509-9-S1-S9], and detecting MiRNA-disease associations [@doi:10.1155/2017/2498957].
+In this section we discuss on multi-omic applications that utilize projecting knowledge graphs into a low dimensional space to make connections.
+
+Knowledge graphs have been used as recommendation systems to establish links between RNA with disease and proteins with other proteins.
+Shen et al. [@doi:10.1155/2017/2498957] used an algorithm called collaborative filtering to establish an association between MiRNA and diseases.
+The authors constructed an MiRNA-Disease network using the HMDDD database [@doi:10.1093/nar/gky1010] and generated an adjacency matrix with the rows representing MiRNA and the columns representing diseases.
+This adjacency matrix was decomposed into small rectangular matrices using SVD, then these matricies were used to calculate similarity scores between the MiRNA and diseases.
```

Comment:
It is hard to use the github interface to make each change. MiRNA should be miRNA. The proper noun is lower cased because the portmanteau of micro RNA was initially done that way.